### ISSUE TITLE
CodePush app startup time issues

### ISSUE THREAD

**Issue by bulby97** [microsoft/react-native-code-push#1794]
### Steps to Reproduce
1. Change one line of code - deploy code-push update
2. Open App just installed on phone - App loads quickly
3. Wait for update to download, close App
3. Open App - App loads slowly

### Expected Behavior
The startup time between the fresh app and the updated app should be the same

### Actual Behavior
Startup time on updated app is very slow. I don't really understand why since it's just one line of code that has been changed. I'm using Hermes. It seems like the new javascript bundle is really heavy to process ?

### Screenshots 
![Feb-18-2020 21-27-49](https://user-images.githubusercontent.com/3509412/74717941-26568d80-5296-11ea-9212-ac0a0f678288.gif)

### Environment

* react-native-code-push version: 6.1.0
* react-native version: 0.61.4
* iOS/Android/Windows version: Android 10 / Android 9
* Does this reproduce on a debug build or release build? Release
* Does this reproduce on a simulator, or only on a physical device? Both

**Comment by Krasavinigor** [comment]
Hello @bulby97!

Unfortunately, I couldn't reproduce this issue. I created a demo project and added Codepush plugin. I've got the same startup time before and after updates.
Could you please provide a test project with this issue?

Thank you!

**Comment by alexandergoncharov** [comment]
Hi @bulby97 ,

I'm going to close this issue for now as we haven't heard from you in a while, unfortunately.
Please feel free to reopen it if you have any questions.

**Comment by alihamza1729** [comment]
I am facing same issue 

my react native version:0.72.7
codePush:8.1.0

**Comment by alihamza1729** [comment]
> ### Steps to Reproduce
> 1. Change one line of code - deploy code-push update
> 2. Open App just installed on phone - App loads quickly
> 3. Wait for update to download, close App
> 4. Open App - App loads slowly
> 
> ### Expected Behavior
> The startup time between the fresh app and the updated app should be the same
> 
> ### Actual Behavior
> Startup time on updated app is very slow. I don't really understand why since it's just one line of code that has been changed. I'm using Hermes. It seems like the new javascript bundle is really heavy to process ?
> 
> ### Screenshots
> ![Feb-18-2020 21-27-49](https://user-images.githubusercontent.com/3509412/74717941-26568d80-5296-11ea-9212-ac0a0f678288.gif) [ ![Feb-18-2020 21-27-49](https://user-images.githubusercontent.com/3509412/74717941-26568d80-5296-11ea-9212-ac0a0f678288.gif) ](https://user-images.githubusercontent.com/3509412/74717941-26568d80-5296-11ea-9212-ac0a0f678288.gif) [ ](https://user-images.githubusercontent.com/3509412/74717941-26568d80-5296-11ea-9212-ac0a0f678288.gif)
> 
> ### Environment
> * react-native-code-push version: 6.1.0
> * react-native version: 0.61.4
> * iOS/Android/Windows version: Android 10 / Android 9
> * Does this reproduce on a debug build or release build? Release
> * Does this reproduce on a simulator, or only on a physical device? Both

Did you resolve it ?

**Comment by mk04366** [comment]
Facing this same issue on both Android and IOS

```
"react-native": "0.73.6",
"react-native-code-push": "^8.2.2",
```

Was anyone able to resolve this issue?

**Comment by mk04366** [comment]
If someone is still facing this issue, try this fix: [#2448](https://github.com/microsoft/react-native-code-push/issues/2448#issuecomment-1683473610). Worked for me
